modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-restricted-network_{context}"]
+= Enabling your Operator for restricted network environments
+
+As an Operator author, your CSV must meet additional requirements for your
+Operator to run properly in {product-title} clusters that are in a restricted
+network environment:
+
+* List any _related images_, or other container images that your Operator might
```

Comment:
Just wondering if it's really necessary for this to be emphasized like this

Reply:
Just as the first appearance of the term. Though it is fairly self-explanatory.